sub/source/docs/arming_the_motors.rst

```diff
@@ -0,0 +1,29 @@
+.. _arming_the_motors:
+
+=================
+Arming the motors
+=================
+
+Arming the vehicle allows the thrusters to start spinning.  Before arming,
+make sure all people, objects, and any body parts (e.g., hands) are clear of the thruster propellers.
+
+In order to arm:
+
+- Pre-Arm safety checks must be passing (See :ref:`common-prearm-safety-checks`). It is recommended that :ref:`ARMING_CHECK<ARMING_CHECK>` be enabled for all available systems/sensors checks to prevent arming with system issues present. You must have also provided a method to Disarm the vehicle, either by GCS button assignments (``BTNx_FUNCTION``), or :ref:`AUX_FUNCTION <common-auxiliary-function>`.
+- The throttle must be at idle value if :ref:`RC_OPTIONS<RC_OPTIONS>` bit 5 is set ("Arming check throttle for 0 input"). Normally, this would be a center stick if bi-directionaa vertical thrusters are used.
```

Comment:
```suggestion
- The throttle must be at idle value if :ref:`RC_OPTIONS<RC_OPTIONS>` bit 5 is set ("Arming check throttle for 0 input"). Normally, this would be a center stick if bi-directional vertical thrusters are used.
```

Reply:
thanks